app/pktgen-rate.c

```diff
@@ -307,7 +307,7 @@ pktgen_page_rate(void)
     unsigned sp;
     char buff[32];
     int display_cnt;
-    uint64_t avg_lat, ticks;
+    uint64_t avg_lat, ticks, max_lat;
```

Comment:
This change does not set max_lat == 0.

Reply:
That's not required as there is no use of max_lat till line 388 and on line 379 we are assigning it a value 0.